hotshot-state-prover/src/service.rs

```diff
@@ -79,6 +79,12 @@ pub struct StateProverConfig {
     pub stake_table_capacity: usize,
 }
 
+#[inline]
+/// A helper function to compute the quorum threshold given a total amount of stake.
+pub fn compute_quorum_threshold(total_stake: U256) -> U256 {
```

Comment:
I think we should call this `one_honest_threshold`, not quorum

Reply:
Right: [0a6b8b1](https://github.com/EspressoSystems/espresso-sequencer/pull/1461/commits/0a6b8b1d97294c7b3625836a6c33bec03003b261)